sdk/identity/azure-identity/src/main/java/com/azure/identity/AuthenticationRecord.java

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.aad.msal4j.IAuthenticationResult;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+/**
+ * Represents the account information relating to an authentication request
+ */
+public class AuthenticationRecord {
+    private final static ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    @JsonProperty("authority")
+    private String authority;
+
+    @JsonProperty("homeAccountId")
+    private String homeAccountId;
+
+    @JsonProperty("tenantId")
+    private String tenantId;
+
+    @JsonProperty("username")
+    private String username;
+
+
+    AuthenticationRecord() { }
```

Comment:
Why is this ctor needed? This is an immutable class and only the other ctor overload is sufficient.

Reply:
needed for Jackson deserialize.